### DISCUSSION TITLE
Seeeduino Nano | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#1099] [Q&A]
# Seeeduino Nano | Seeed Studio Wiki

Seeeduino Nano

https://wiki.seeedstudio.com/Seeeduino-Nano/

**Comment by HW-Dude67** [comment]
Does VDD need to be connected to VCC? There does not seem to be a source for VDD.

**Reply by HW-Dude67** [reply]
Nope. There is an internal reg on the CP2102. The input to the reg is on pin7 (VREGIN)
